pkg/cmd/config/init/config_init.go

```diff
@@ -22,8 +22,10 @@ import (
 
 type ConfigInitOpt struct {
 	EnvironmentsRepo   string
-	File               string
-	RepositoriesDir    string
+	ConfigDir          string
+	ConfigFile 	       string
+	InitFile           string
```

Comment:
I spent a fair bit of time being confused about what we should and shouldn't have in here.
From a user perspective, they want an experience roughly similar to kubernetes `kube-config.yaml`
However, that experience has one file, and multiple connections in the file, where one is listed as currently active.
We're not gonna go that far, we're just gonna have a directory, and the config file/logs/repositories are stored in there.

User should either specify "I want to use this directory" by supplying an environment variable, or perhaps the CLI should just use whatever config is found in the current directory